test/README

```diff
@@ -24,17 +24,14 @@ for more details and recommendations on ensuring it is enabled correctly.
 
 ## Introduction
 
-Before running the tests, ensure Cockpit has been built where the test suite
-expects to find it (do NOT run the build step as root):
+To build Cockpit and run the integration tests run the following (do NOT run
+the integration tests as root):
 
-   $ ./test/verify/testsuite-prepare
+    $ ./test/verify/run-tests --install
```

Comment:
I'm not sure I like that. I would like to keep documenting it as two separate steps. It's unnerving and absolutely unnecessary to rebuild the test VM (i. e. the `testsuite-prepare` part) when working on the tests, i. e. the installed cockpit doesn't change. It's of course fine to point out that running `run-tests` or any `check-*` with `--install` is a shortcut for this step.